pkg/api/types.go

```diff
@@ -2785,6 +2802,12 @@ type DeleteOptions struct {
 	// finalizer will be added to/removed from the object's finalizers list.
 	// +optional
 	OrphanDependents *bool
+
+	// Whether and how garbage collection will be performed.
+	// Defaults to DeletePropagationDefault.
+	// Either this field or OrphanDependents may be set, but not both.
+	// +optional
+	PropagationPolicy *DeletePropagationPolicy
```

Comment:
Oh, I see DeletePropagationPolicy is an alias to string. I will look into this, it should be a bug.